Tests/IntegrationTests/PostgresNIOTests.swift

```diff
@@ -707,7 +707,7 @@ final class PostgresNIOTests: XCTestCase {
         XCTAssertThrowsError(
             try PostgresConnection.connect(
                 to: SocketAddress.makeAddressResolvingHost("elmer.db.elephantsql.com", port: 5432),
-                tlsConfiguration: .forClient(certificateVerification: .fullVerification),
+                tlsConfiguration: .makeClientConfiguration(),
```

Comment:
We seem to not be passing the certificate verification in the new version?

Reply:
`.fullVerification` is the default for `.makeClientConfiguration()`